crates/red_knot_python_semantic/src/module_resolver/resolver.rs

```diff
@@ -18,7 +18,7 @@ use super::path::{ModulePath, SearchPath, SearchPathValidationError};
 
 /// Resolves a module name to a module.
 pub fn resolve_module(db: &dyn Db, module_name: &ModuleName) -> Option<Module> {
-    let interned_name = ModuleNameIngredient::new(db, module_name);
+    let interned_name = ModuleNameIngredient::new(db, module_name.clone());
```

Comment:
This change was caused by the new bounds introduced in https://github.com/salsa-rs/salsa/commit/f428a94a9f350d35ae5c68c6e1e11db7fe74d87a. I'm not sure why there's no blanket implementation for `&T` anymore.

Reply:
Upstream PR https://github.com/salsa-rs/salsa/pull/678